tests/async_client_test.py

```diff
@@ -370,6 +370,21 @@ def test_register(self, async_client, aioresponse):
         assert isinstance(resp, RegisterResponse)
         assert async_client.access_token
 
+    def test_deactivate(self, async_client, aioresponse):
+        loop = asyncio.get_event_loop()
+
+        assert not async_client.access_token
+
+        aioresponse.post(
+            "https://example.org/_matrix/client/r0/admin/deactivate",
+            status=200,
+            payload=self.DeactivateResponse
```

Comment:
This doesn't exist, you need to create a fake response/json dict with your response.

Reply:
I thought I did. It must have not been git-added. Will work on that.

Reply:
Ah, it got lost in the merge conflicts. Re-adding.